include/intx/intx.hpp

```diff
@@ -1785,7 +1775,7 @@ inline void udivrem_knuth(
 }  // namespace internal
 
 template <unsigned M, unsigned N>
-div_result<uint<M>, uint<N>> udivrem(const uint<M>& u, const uint<N>& v) noexcept
+constexpr div_result<uint<M>, uint<N>> udivrem(const uint<M>& u, const uint<N>& v) noexcept
```

Comment:
Was this `constexpr` just missed?

Reply:
Yes. Maybe we never used it in constexpr context but now compiler is not happy as we removed one level of templates and maybe it is obviously wrong now.